charts/unleash-edge/values.yaml

```
@@ -0,0 +1,94 @@
+## Default values for unleash-edge
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: unleashorg/unleash-edge
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose defualt is the chart appVersion
```

Thanks github for finding typos 🙌🏼 

```suggestion
  # Overrides the image tag whose default is the chart appVersion
```